_episodes_rmd/14-tidyr.Rmd

```diff
@@ -142,6 +142,8 @@ The first step towards getting our nice intermediate data format is to first
 convert from the wide to the long format. The `tidyr` function `gather()` will
 'gather' your observation variables into a single variable.
 
+![](../fig/14-tidyr-fig3.svg)
```

Comment:
There are now two figures named fig3 in this lesson (there's another one on line 165/167). Should probably renumber that one and make sure it can be generated from the scripts earlier.